snuba/clusters/cluster.py

```diff
@@ -335,6 +357,10 @@ def __get_cluster_nodes(self, cluster_name: str) -> Sequence[ClickhouseNode]:
     if (s not in DEV_STORAGE_SETS or settings.ENABLE_DEV_FEATURES)
 }
 
+assert (
+    not expected_storage_sets - _unique_registered_storage_sets
+), "All storage sets must be assigned to a cluster"
```

Comment:
Why do you need to add this back? I thought we recently removed it for some reason.

Reply:
I think I branched off an old master maybe -- I'll keep in mind we don't want to restore it